### PR TITLE
JBIDE-20697 tools.jboss.org/features: Add JavaEE Batch

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: features/batch.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+:page-feature_order: 8
+:page-feature_tagline: JavaEE 7 Batch Processing
+:page-feature_image_url: images/batch_icon_256px.png
+:page-issues_url: https://issues.jboss.org/browse/JBIDE/component/12320844
+
+== JavaEE 7 Batch Processing
+=== Makes using Batch easier
+image::images/features-batch.png[]
+
+Batch Tools helps you with developing Java applications which use Batch Processing programming model.
```
Comment (mishaone) [inline features/batch.adoc]: Original: Batch Tools helps you with developing Java applications which use Batch Processing programming model. It provides you with New Job XML and New Batch Artifact wizards. Advanced visual and source editors for Job XML files. Project validation.

Proposed: Batch Tools help users develop Java applications that use the Batch Processing programming mode. Batch Tools also include features such as New Job XML, New Batch Artifact Wizards, Advanced Visual and Source Editors for Job XML files, as well as project validation.

File: features/batch.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+== JavaEE 7 Batch Processing
+=== Makes using Batch easier
+image::images/features-batch.png[]
+
+Batch Tools helps you with developing Java applications which use Batch Processing programming model. 
+It provides you with New Job XML and New Batch Artifact wizards. Advanced visual and source editors for Job XML files. Project validation.
+
+== Wizards
+=== New Job XML file. New Bach Artifacts
```
Comment (mishaone) [inline features/batch.adoc]: minor typo: s/New Bach Artifacts/New Batch Artifacts

**Comment by mishaone** [comment]
Two suggested changes, otherwise looks good to me.

**Comment by alexeykazakov** [comment]
Thanks!. Pushed to master and production.

**Pull request closed without merging**